pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1192,19 +1197,27 @@ func encodeSystemAdapterInfo(dpcl types.DevicePortConfigList) *info.SystemAdapte
 
 		dps.Ports = make([]*info.DevicePort, len(dpc.Ports))
 		for j, p := range dpc.Ports {
-			dps.Ports[j] = encodeNetworkPortConfig(&p)
+			dps.Ports[j] = encodeNetworkPortConfig(ctx, &p)
 		}
 		sainfo.Status[i] = dps
 	}
 	log.Debugf("encodeSystemAdapterInfo: %+v\n", sainfo)
 	return sainfo
 }
 
-func encodeNetworkPortConfig(npc *types.NetworkPortConfig) *info.DevicePort {
+func encodeNetworkPortConfig(ctx *zedagentContext,
+	npc *types.NetworkPortConfig) *info.DevicePort {
+	aa := ctx.assignableAdapters
+
+	ibPtr := aa.LookupIoBundlePhylabel(npc.Phylabel)
+
 	dp := new(info.DevicePort)
 	dp.Ifname = npc.IfName
 	// XXX rename the protobuf field Name to Phylabel and add Logicallabel?
 	dp.Name = npc.Phylabel
+	if ibPtr != nil {
+		dp.Usage = info.InfoPhyIoMemberUsage(ibPtr.Usage)
+	}
```

Comment:
What should we say if we have no ibPtr? "none" and "unknown" might be an important distinction when things are confused.

Reply:
InfoPhyIoUsageNone will be the value if ibPtr == nil.

If we see this value - we can take it that ib is unknown. I don't think there is much value in adding a separate InfoPhyIoUsageUnknown

Reply:
OK